src/reduce.js

```diff
@@ -0,0 +1,15 @@
+import walk from "./walk.js";
+
+/**
+ * Reduce a tree to a single value
+ * @param {object} node
```

Comment:
Will JSDoc complain if we use `{Node}` in these?

Reply:
Forgot to respond to this, but I can check; `npm run build:docs` is currently not working for some reason so I'm trying to debug it before I can check whether or not this would work

Reply:
![Screen Shot 2024-03-16 at 8 07 37 AM](https://github.com/mavoweb/treecle/assets/45052692/720b56c9-2421-4f6f-8294-2581fe7c9557)

Looks like it works!